### PR TITLE
[CARBONDATA-2222] Update the FAQ doc for some mistakes

### PR DESCRIPTION
**Pull request by chenerlu**
Be sure to do all of the following checklist to help us incorporate 
your contribution quickly and easily:

 - [No] Any interfaces changed?
 
 - [No] Any backward compatibility impacted?
 
 - [Yes ] Document update required?

 - [NA ] Testing done
        Please provide details on 
        - Whether new unit test cases have been added or why no new tests are required?
        - How it is tested? Please attach test report.
        - Is it a performance related change? Please attach the performance test report.
        - Any additional information to help reviewers in testing this change.
       
 - [ NA] For large changes, please consider breaking it into sub-tasks under an umbrella JIRA.

### CODE REVIEW

**Comment by ravipesala** [comment]
SDV Build Success , Please check CI http://144.76.159.231:8080/job/ApacheSDVTests/3760/

**Comment by CarbonDataQA** [comment]
Build Success with Spark 2.1.0, Please check CI http://136.243.101.176:8080/job/ApacheCarbonPRBuilder1/4066/

File: docs/faq.md [hunk 1]
```diff
@@ -143,38 +143,38 @@ INSERT INTO TABLE carbon_table SELECT id, city FROM source_table;
...
 When SubQuery predicate is present in the query.
 
-Example 
+Example:
 
 ```
-create table gdp21(cntry smallint, gdp double, y_year date) stored by 'carbondata'
-create datamap ag1 on table gdp21 using 'preaggregate' as select cntry, sum(gdp) from gdp group by ctry;
-select ctry from pop1 where ctry in (select cntry from gdp21 group by cntry)
+create table gdp21(cntry smallint, gdp double, y_year date) stored by 'carbondata';
```
Comment (jackylk) [inline docs/faq.md]: remove `;`

**Comment by chenliang613** [comment]
LGTM

**Pull request closed without merging**